### DISCUSSION TITLE
Windows 11

### DISCUSSION THREAD

**Discussion by rlnt** [Sophia-Community/SophiApp#10] [Q&A] [answered]
Looks really promising. Do you plan to add Windows 11 support too in the future?
I am on Windows 11 21H2. Interestingly, the `SophieApp-XX.log` says my OS is `Windows 10 Pro 21H2 build: 10.0.22000.318`.

**Comment by farag2** [comment, marked as answer]
Of course! With the 2.0 release with new UI. In 2022 ;)

**Comment by farag2** [comment]
We want to go further, to the Fluent UI design. PowerToys is a very good example of how a Windows app with Fluent UI should looks like.

**Reply by rlnt** [reply]
Not gonna lie, I am a daily user of PowerToys and I love it.
Every time I set up a new OS, Sophia is the first thing to run. The UI could be very nice to also change settings later on especially when features are implemented later on.

How will it work when it's ready for action? Will it detect the current settings and then you can just switch them in the app? Or will you configure the script and then run it like the already existing Wrapper does?

**Reply by farag2** [reply]
It's already determening the current settings. That's why it takes time to load the app. So the state of every function reflects the current "situation" on OS.
SophiApp is script independent app. It doesn't work as David's Wrapper.

**Reply by rlnt** [reply]
Awesome. Exactly what I hoped for. Thanks for the clarification. Can't wait for the full release and Windows 11 support. No pressure though 😋
